source/http_connection.c

```diff
@@ -14,33 +14,37 @@
  */
 
 #include "http_connection.h"
+#include "io.h"
 #include "module.h"
 #include "uv_interop.h"
 
 #include <aws/http/connection.h>
 #include <aws/io/tls_channel_handler.h>
 
-struct http_nodejs_connection {
+struct http_connection_binding {
     struct aws_http_connection *connection;
+    napi_value node_external;
     struct aws_tls_connection_options tls_options;
```

Comment:
there's no need to keep the `aws_tls_connection_options` struct as a permanent member. It's all copied when aws_http_client_connect() is called.

Reply:
oh fabulous, will fix.